app/templates/posts.html

```diff
@@ -1,6 +1,6 @@
 ﻿{% extends "base.html" %}
```

Comment:
See comment under `deleteClass.html`.
```suggestion
﻿{% from "_formhelpers.html" import render_error %}
﻿{% extends "base.html" %}
```